File: app/src/main/java/com/example/coffeehelper/BrewSelectorActivitiy.java
package com.example.coffeehelper;

import android.app.Activity;
import android.app.AlertDialog;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class BrewSelectorActivitiy extends Activity {
    public void onCreate(Bundle savedState) {
        super.onCreate(savedState);
        setContentView(R.layout.activity_selector);

        // Set up three buttons for layout
        Button btnEspresso = (Button) findViewById(R.id.btnEspresso);
        Button btnFrenchPress = (Button) findViewById(R.id.btnFrenchPress);
        Button btnDrip = (Button) findViewById(R.id.btnDrip);

        btnEspresso.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

            }
        });

        btnFrenchPress.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

            }
        });

        btnDrip.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
            }
        });
    }
}

File: app/src/main/java/com/example/coffeehelper/BrewSelectorActivitiy.kt
package com.example.coffeehelper

import android.app.Activity
import android.os.Bundle
import android.view.View
import android.widget.Button

class BrewSelectorActivitiy : Activity() {
    public override fun onCreate(savedState: Bundle?) {
        super.onCreate(savedState)
        setContentView(R.layout.activity_selector)

        // Set up three buttons for layout
        val btnEspresso = findViewById<View>(R.id.btnEspresso) as Button
        val btnFrenchPress = findViewById<View>(R.id.btnFrenchPress) as Button
        val btnDrip = findViewById<View>(R.id.btnDrip) as Button
        btnEspresso.setOnClickListener { }
        btnFrenchPress.setOnClickListener { }
        btnDrip.setOnClickListener { }
    }
}